Source/ALS/Private/AlsCharacter.cpp

```diff
@@ -991,12 +996,12 @@ FGameplayTag AAlsCharacter::CalculateActualGait(const FGameplayTag& MaxAllowedGa
 	// different from the desired gait or max allowed gait. For instance, if the max allowed gait becomes
 	// walking, the new gait will still be running until the character decelerates to the walking speed.
 
-	if (LocomotionState.Speed < AlsCharacterMovement->GetGaitSettings().WalkSpeed + 10.0f)
+	if (LocomotionState.Speed < AlsCharacterMovement->GetWalkSpeed() + 10.0f)
```

Comment:
I think this is wrong. You should use `WalkForwardSpeed` instead.

Reply:
Is means calculate the gait alway use the forwad speed, even if the actor is walking backward?